src/main/java/com/hedera/hashgraph/sdk/Client.java

```diff
@@ -40,15 +37,6 @@ public Client(Map<AccountId, String> nodes) {
             .collect(Collectors.toUnmodifiableMap(Map.Entry::getKey, t -> new Node(t.getKey(), t.getValue())));
     }
 
-    public Client setMaxTransactionFee(@Nonnegative long maxTransactionFee) {
-        if (maxTransactionFee <= 0) {
-            throw new IllegalArgumentException("maxTransactionFee must be > 0");
-        }
-
-        this.maxTransactionFee = maxTransactionFee;
-        return this;
-    }
-
     public Client setOperator(AccountId operatorId, Ed25519PrivateKey operatorKey) {
```

Comment:
Are these mainly changes from reformatting the code based on platform's intellij rules?

Reply:
Yes, I can undo if you like. It just auto-saves that way.

Reply:
Nope, all good. Long as we can say that to LB on the main PR. If they want it undone, we'll do that then.